docs/pages/color_palette.js

```diff
@@ -0,0 +1,98 @@
+// @flow strict
+import type { Node } from 'react';
+import { Flex } from 'gestalt';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import ColorPalette from '../components/ColorPalette.js';
+import ColorTile from '../components/ColorTile.js';
+import Page from '../components/Page.js';
+
+const colors = [
+  { name: 'Pushpin', id: 'red', textColor: 'white' },
+  { name: 'Flaminglow', id: 'pink', textColor: 'darkGray' },
+  { name: 'Skycicle', id: 'blue', textColor: 'darkGray' },
+  { name: 'Spabattical', id: 'teal', textColor: 'darkGray' },
+  { name: 'Matchacado', id: 'green', textColor: 'darkGray' },
+  { name: 'Mysticool', id: 'purple', textColor: 'white' },
+  { name: 'Firetini', id: 'orange', textColor: 'darkGray' },
+  { name: 'Caramellow', id: 'yellow', textColor: 'darkGray' },
+];
+
+const neutrals = [
+  { name: 'Mochimalist', id: 'white', textColor: 'darkGray' },
+  { name: 'Roboflow', id: 'gray', textColor: 'darkGray' },
+  { name: 'Cosmicore', id: 'black', textColor: 'white' },
+];
+
+export default function ColorPage(): Node {
+  return (
+    <Page title="Color Palette">
+      <PageHeader
+        name="Color palette"
+        description={`
+        Our color palettes are shared between Brand and Gestalt, and represent our full range of options. The colors are divided into baseline, extended, and reserved colors.
+
+        For further Brand guidance, please reference the [Brand guidelines website](https://brand.pinterest.com/).
+        `}
+        showSourceLink={false}
+      />
+      <MainSection
+        name="Baseline colors"
+        description={`
+        Our baseline palette helps to create consistency across products.
+        The baseline palette is comprised of our hero and neutrals colors, allowing the Pinterest content to shine, while ensuring enough color contrast. In addition, these colors should be purpose-driven to provide a better user experience. Check out [Color Usage](/color_usage) for guidance.
```

Comment:
not sure if it's a term of art, but "these colors should be purpose-driven to provide a better user experience" doesn't mean much to me?

Reply:
Updated the phrasing here